src/app/features/create-event/event-details-form/event-details-form.component.ts

```diff
@@ -17,20 +21,34 @@ export class EventDetailsFormComponent implements OnInit, OnDestroy {
   private createEventForm = inject(CreateEventFormService);
   private createEventService = inject(CreateEventService);
   private errorService = inject(ErrorhandlerService);
-  private unsubscribe$$ = new Subject<void>();
 
-  constructor() {
-    this.eventTypeForm = this.createEventForm.getForm().controls.eventTypeForm;
-    this.eventDetailsForm = this.createEventForm.getForm().controls.eventDetailsForm;
-  }
+  private unsubscribe$$ = new Subject<void>();
+  private allHashtags: string[] = [];
 
-  categories$ = this.createEventService.getAllCategories();
   errorClientServer$ = this.errorService.error$;
+  categories$ = this.createEventService.fetchAllCategories();
+  allHashtags$ = this.createEventService.fetchAllHashtags();
+  groups$ = this.createEventService.fetchUserGroups();
+
+  hashtagCtrl = new FormControl('', {
+    validators: [
+      Validators.maxLength(20),
+      Validators.minLength(2),
+      Validators.pattern(pattern.lettersNumbersDashesAndPolishLettersRegex),
+    ],
+  });
+  filteredHashtags$?: Observable<string[]>;
+  hashtags: string[] = [];
+  separatorKeysCodes: number[] = [ENTER, COMMA];
   eventTypeForm: FormGroup<EventTypeForm>;
   eventDetailsForm: FormGroup<EventDetailsForm>;
   today = new Date();
   hourMatcher = new HourErrorStateMatcher();
-  groups$ = this.createEventService.fetchUserGroups();
+
+  constructor() {
+    this.eventTypeForm = this.createEventForm.getForm().controls.eventTypeForm;
+    this.eventDetailsForm = this.createEventForm.getForm().controls.eventDetailsForm;
```

Comment:
możemy to zrobić bezpośrednio w przypisaniu pola, nie potrzebujemy tego robić w constructorze